server/src/main/java/org/elasticsearch/plugins/PluginInfo.java

```diff
@@ -49,6 +50,8 @@
     public static final String ES_PLUGIN_PROPERTIES = "plugin-descriptor.properties";
     public static final String ES_PLUGIN_POLICY = "plugin-security.policy";
 
+    private static final Version QUOTA_FS_PLUGIN_SUPPORT = Version.CURRENT;
```

Comment:
This should be 7.11?

Reply:
I can't do that until the changes are backported - I learnt that the hard way on another branch.

Reply:
This can be done. There are 2 ways to handle the backports. One is to put CURRENT in master, change it on the backport, then go back to master and change all the occurrences. The other is to put the intended version in master and disable bwc tests (there is a global flag in the root build.gradle), backport, then re-enable bwc tests in master after the backport is complete. The latter has the advantages that the backport can be tested locally before committing to master, as well as only a single place to change (the bwc flag) on followup, instead of potentially multiple versioned conditional statements.

Reply:
I believe this is the only place that I reference a version. I'll go for the former route I think.